bumblebee_status/modules/core/pulsectl.py

```diff
@@ -65,6 +65,9 @@ def __init__(self, config, theme, type):
         self.__show_device_name = util.format.asbool(
             self.parameter("showdevicename", False)
         )
+        self.__popup_font_size = util.format.asint(
+            self.parameter("popup_font_size", None), 0
+        )
```

Comment:
For now, I've only added the font size parameter to this module, but it should probably be added to all the ones that use the popup menu